pkg/resource/resource.go

```diff
@@ -215,11 +163,15 @@ func (a *Attributes) GetString(path string) *string {
 }
 
 func (a *Attributes) GetStringSlice(path string) []string {
-	val, exist := (*a)[path]
-	if !exist {
+	val := a.GetSlice(path)
+	if val == nil {
 		return nil
 	}
-	return val.([]string)
+	slice := make([]string, 0, len(val))
```

Comment:
Why do we need to make a copy ?

Reply:
Just like the `GetMap` we can't use the `[]string` cast on `[]interface{}`. I don't remember the resource I got the error ..

Reply:
same as getStringmap it is not possible to cast []inteferace{} to []string